linera-base/src/identifiers.rs

```diff
@@ -378,6 +379,12 @@ impl From<Ed25519PublicKey> for AccountOwner {
     }
 }
 
+impl From<EvmPublicKey> for AccountOwner {
+    fn from(public_key: EvmPublicKey) -> Self {
+        AccountOwner::Address32(CryptoHash::new(&public_key))
```

Comment:
I believe this is where we should create a 20-byte address in the Ethereum way, so Linera applications show the same user address as Metamask

Reply:
Done in [EvmPublicKey maps to AccountOwner::Address20](https://github.com/linera-io/linera-protocol/pull/3674/commits/c981041ff3578149275cd15b2b17c07d91ed3fc2)